sdk/cosmos/azure-cosmos/azure/cosmos/aio/cosmos_client.py

```diff
@@ -109,8 +113,13 @@ class CosmosClient(object):
             :name: create_client
     """
 
-    def __init__(self, url, credential, consistency_level=None, **kwargs):
-        # type: (str, Any, Optional[str], Any) -> None
+    def __init__(
+            self,
+            url: str,
+            credential: Any,
```

Comment:
The use of `Any` is discouraged as it does not provide much (if any?) value. This should be the class of your credential object.